libsolidity/formal/Predicate.cpp

```diff
@@ -163,18 +163,22 @@ string Predicate::formatSummaryCall(vector<string> const& _args) const
 
 	/// The signature of a function summary predicate is: summary(error, this, cryptoFunctions, txData, preBlockChainState, preStateVars, preInputVars, postBlockchainState, postStateVars, postInputVars, outputVars).
 	/// Here we are interested in preInputVars.
-	vector<string>::const_iterator first = _args.begin() + 5 + static_cast<int>(stateVars->size());
-	vector<string>::const_iterator last = first + static_cast<int>(fun->parameters().size());
+	vector<smtutil::Expression>::const_iterator first = _args.begin() + 5 + static_cast<int>(stateVars->size());
```

Comment:
Hm, I think this is a good case for using `auto`? Iterators have unnecessarily long type names